src/status_im/ui/screens/hardwallet/setup/views.cljs

```diff
@@ -288,20 +292,34 @@
     [react/activity-indicator {:animating true
                                :size      :large}]]])
 
-(defn- preparing []
-  [react/view styles/loading-view-container
-   [react/view styles/center-container
-    [react/text {:style styles/center-title-text
-                 :font  :bold}
-     (i18n/label :t/preparing-card)]
-    [react/text {:style           styles/generating-codes-for-pairing-text
-                 :number-of-lines 2}
-     (i18n/label :t/generating-codes-for-pairing)]
-    [react/text {:style styles/estimated-time-text}
-     (i18n/label :t/taking-long-hold-phone-connected)]]
-   [react/view styles/waiting-indicator-container
-    [react/activity-indicator {:animating true
-                               :size      :large}]]])
+(defview preparing []
+  (letsubs [progress-bar (reagent/atom nil)
```

Comment:
better to use clj atom , not reagent atom, because you don't need reactions

Reply:
good point, changed